staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -273,12 +273,29 @@ func (a *GenericAdmissionWebhook) callHook(ctx context.Context, h *v1alpha1.Exte
 		return nil
 	}
 
-	if response.Status.Result == nil {
-		return fmt.Errorf("admission webhook %q denied the request without explanation", h.Name)
+	return toStatusErr(h.Name, response.Status.Result)
```

Comment:
Perhaps, instead of calling this here, we should find each call site of callHook and pass any error it returns through toStatusErr?

Reply:
What about `*ErrCallingWebhook`? It's handled differently in `Admit`.